backend/src/app.ts

```diff
@@ -1,10 +1,12 @@
 import express, { Express } from 'express';
 import morgan from 'morgan';
 import cors from 'cors';
-// Import controllers
```

Comment:
you didn't update this comment.

// Import controllers ----->> // Import API routes